exchanges/coinbene/coinbene_wrapper.go

```diff
@@ -56,8 +55,11 @@ func (c *Coinbene) SetDefaults() {
 	c.CurrencyPairs = currency.PairsManager{
 		AssetTypes: asset.Items{
 			asset.Spot,
+			asset.PerpetualSwap,
```

Comment:
not a nit but something I noticed: On first run with upgrade from spot to include perpetual swap in the config. In manager types enabled and available have json tags omitempty; maybe take them out, on second run it seems to set enabled to a random pair anyway, like it should.

Reply:
Good point, have pushed an update to remove the omitempty's. There still exists a caveat WRT to our loading of config -> exchange SetDefaults() changes -> config upgrades as it is a two step process and is something which will require an overhaul of config/exchange pair loading